src/main/java/net/sf/jabref/gui/maintable/MainTableColumn.java

```diff
@@ -144,4 +132,26 @@ public JLabel getHeaderLabel() {
             return new JLabel(getDisplayName());
         }
     }
+
+    public boolean isResolved(BibEntry entry) {
```

Comment:
I think, that one needs the information on a per-field, not on a per-entry basis. As far as I understood, I get it for both cases: resolved fields are highlighted and there is an indication on the complete entry. - This is OK :innocent: